src/electron/routing_service.ts

```diff
@@ -107,7 +113,7 @@ export class RoutingDaemon {
       });
 
       const initialErrorHandler = () => {
-        if (!(isLinux && retry)) {
+        if (!retry) {
```

Comment:
Why are we removing the linux check?

Reply:
This change enables the routing service restart mechanism to work on Windows, not only Linux.